plottable-npm.d.ts

```diff
@@ -1665,6 +1665,7 @@ declare namespace Plottable.Components {
     }
 }
 declare namespace Plottable {
+    type AxisOrientation = "left" | "top" | "right" | "bottom";
```

Comment:
not currently the standard in this library so perhaps not worth it but I think it would be worth alphabetizing things   like this